.envrc

```diff
@@ -0,0 +1,2 @@
+# Direnv configuration
```

Comment:
This file is for https://github.com/nix-community/nix-direnv / direnv.net to load the shell once you `cd` into the directory. Requires first-time approval.